securesystemslib/keys.py

```diff
@@ -716,7 +717,9 @@ def create_signature(key_dict, data):
     sig, scheme = securesystemslib.ed25519_keys.create_signature(
         public, private, data, scheme)
 
-  elif keytype == 'ecdsa-sha2-nistp256':
+  # Continue to support keytypes of ecdsa-sha2-nistp256 and ecdsa-sha2-nistp384
+  # for backwards compatibility with older securesystemslib releases
+  elif keytype in ['ecdsa', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384']:
```

Comment:
NOTE: We didn't support generation of `ecdsa-sha2-nistp384` signatures prior to this PR (https://github.com/secure-systems-lab/securesystemslib/pull/228 only added verification support).

With this patch, we'll fail one level deeper when schemes are checked, which IMO is fine.